Scripts/Python/xOpeningSequence.py

```diff
@@ -223,10 +225,14 @@ def OnGUINotify(self,id,control,event):
                 # see if the there actually is a movie to play
                 skipMovie = 1
                 try:
+                    ageSDL = PtGetAgeSDL()
                     if IsTutorialPath():
-                        os.stat(kAtrusIntroMovie)
+                        gMovieFilePath = kAtrusIntroMovie
+                    elif ageSDL["psnlIntroMovie"]:
```

Comment:
```suggestion
                    elif ageSDL["psnlIntroMovie"][0]:
```
The current code works, but it's somewhat misleading. What we're really testing is the actual value of the variable being non-empty.